README.md

```diff
@@ -15,39 +15,71 @@
   <img src="assets/res.webp"/>
 </p>
 
-# About
-
 This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [Vinceliuice](https://github.com/vinceliuice)
 
 ## Usage
 
 ### Requirements
 
-- GTK `>=3.20`
+- GTK >=3.20
+- `python3`
 - `gnome-themes-extra` (or `gnome-themes-standard`)
 
-### Installation
+To build the theme, make sure the following packages are installed:
+- `sassc`
+- `inkscape`
+- `optipng`
+
+### Installing the theme manually
 
-1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/).
-2. Move the theme folder to **".themes"** in your home directory. **(~/.themes)** (Skip this step if you are using the AUR package)
-3. Select the downloaded theme via your desktop specific tweaks application (**gnome-tweaks** on Gnome 3+).
+1. Download and extract the theme zip from the [latest release](https://github.com/catppuccin/gtk/releases/latest/).
+2. Move the theme folder to the `~/.local/share/themes` directory (skip this step if you're using the AUR package).
+3. Select the downloaded theme via your desktop specific tweaks application (GNOME Tweaks on GNOME 3+).
+4. To theme other apps that are using GTK, make sure to run the following command:
+```bash
+export THEME_DIR="~/.local/share/themes/catppuccin-<flavor>-<accent color>-standard+default"
+mkdir -p "${HOME}/.config/gtk-4.0" && 
+ln -sf "${THEME_DIR}/gtk-4.0/assets" "${HOME}/.config/gtk-4.0/assets" &&
+ln -sf "${THEME_DIR}/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css" &&
+ln -sf "${THEME_DIR}/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
+```
 
-### For Arch Linux users
+###  Applying the theme on Flatpak
+In order for Flatpak to access the theme, make sure to run the following command:
+```bash
+sudo flatpak override --filesystem=$HOME/.local/share/themes
+```
 
-We have 4 AUR packages for all the 4 flavours of the theme:
+Then, run the following command to apply the theme.
+```bash
+sudo flatpak override --env=GTK_THEME=$THEME_DIR
+```
```

Comment:
Keeping it consistent with above.
```suggestion
```bash
export THEME_DIR="~/.local/share/themes/catppuccin-<flavor>-<accent>-standard+default"
sudo flatpak override --env=GTK_THEME=$THEME_DIR
```
```